src/routes/Routes.js

```diff
@@ -1,17 +1,28 @@
-import React from 'react'
+import React, { Suspense, lazy } from 'react'
 import { Redirect, Route, Switch } from 'react-router-dom'
-import Garden from '@components/Garden'
-import Home from '@components/Home'
-import Profile from '@components/Profile/Profile'
+
+import { LoadingRing, Main } from '@1hive/1hive-ui'
+
+const Home = lazy(() => import('@components/Home'))
+const Garden = lazy(() => import('@components/Garden'))
+const Profile = lazy(() => import('@components/Profile/Profile'))
```

Comment:
This is the suggested way to code-split routes in SPAs created using create-react-app or any project which internally uses react-scripts.

Reference link: https://reactjs.org/docs/code-splitting.html#route-based-code-splitting